sql/localsql/0001_migration_test.go

```diff
@@ -16,14 +16,14 @@ import (
 )
 
 func saveNipostChallenge(dir string, ch *types.NIPostChallenge) error {
-	if err := save(filepath.Join(dir, challengeFilename), ch); err != nil {
+	if err := save(filepath.Join(dir, challengeFilename), ch.ToWireV1()); err != nil {
```

Comment:
I think we can remove the 0001 - 0003 migrations with this PR (assuming this change goes into a v1.5.x version of the node).